prometheus_client/bridge/graphite.py

```diff
@@ -92,3 +92,4 @@ def start(self, interval: float = 60.0, prefix: str = '') -> None:
         t = _RegularPush(self, interval, prefix)
         t.daemon = True
         t.start()
+    
```

Comment:
Nit, could you just remove this line so we don't have a needless diff/history entry?

Reply:
Done!